Top-level script command tweaks and README updates (#2071)
ni/nimble: README.md

If you are outside NI, we want to hear from you too! See how to reach out in the [**Community**](#community) section below.

## Getting Started

See the corresponding `Getting Started` section for the general-use Nimble packages:

[![Nimble Angular NPM version and repo link](https://img.shields.io/npm/v/@ni/nimble-angular.svg?label=@ni/nimble-angular)](https://www.npmjs.com/package/@ni/nimble-angular)
[![Nimble Blazor Nuget version and repo link](https://img.shields.io/nuget/v/NimbleBlazor.svg?label=NimbleBlazor)](https://www.nuget.org/packages/NimbleBlazor)
[![Nimble Components NPM version and repo link](https://img.shields.io/npm/v/@ni/nimble-components.svg?label=@ni/nimble-components)](https://www.npmjs.com/package/@ni/nimble-components)

- **[`@ni/nimble-angular`](/angular-workspace/projects/ni/nimble-angular/)** - Handy bindings to the Nimble components for Angular applications.

> - **[`@ni/nimble-blazor`](/packages/nimble-blazor/)** - Handy bindings to the Nimble components for Blazor applications.

Author: ```suggestion
- **[`@ni/nimble-blazor`](/packages/nimble-workspace/)** - Handy bindings to the Nimble components for Blazor applications.
```

> - **[`@ni/nimble-blazor`](/packages/nimble-workspace/)** - Handy bindings to the Nimble components for Blazor applications.

Author: ```suggestion
- **[`@ni/nimble-blazor`](/packages/blazor-workspace/)** - Handy bindings to the Nimble components for Blazor applications.
```

- **[`@ni/nimble-components`](/packages/nimble-components/)** - Nimble components to be used by [any type of  application](https://custom-elements-everywhere.com/).

The context-specific Spright packages:

[![Spright Angular NPM version and repo link](https://img.shields.io/npm/v/@ni/spright-angular.svg?label=@ni/spright-angular)](https://www.npmjs.com/package/@ni/spright-angular)
[![Spright Blazor Nuget version and repo link](https://img.shields.io/nuget/v/SprightBlazor.svg?label=SprightBlazor)](https://www.nuget.org/packages/SprightBlazor)
[![Spright Components NPM version and repo link](https://img.shields.io/npm/v/@ni/spright-components.svg?label=@ni/spright-components)](https://www.npmjs.com/package/@ni/spright-components)

- [`@ni/spright-angular`](/angular-workspace/projects/ni/spright-angular/) - Handy bindings to the Spright components for Angular applications.

> - [`@ni/spright-blazor`](/packages/nimble-blazor/) - Handy bindings to the Spright components for Blazor applications.

Author: ```suggestion
- [`@ni/spright-blazor`](/packages/nimble-workspace/) - Handy bindings to the Spright components for Blazor applications.
```

> - [`@ni/spright-blazor`](/packages/nimble-workspace/) - Handy bindings to the Spright components for Blazor applications.

Author: ```suggestion
- [`@ni/spright-blazor`](/packages/blazor-workspace/) - Handy bindings to the Spright components for Blazor applications.
```

> - **[`@ni/spright-angular`](/angular-workspace/projects/ni/spright-angular/)** - Handy bindings to the Spright components for Angular applications.
> - **[`@ni/spright-blazor`](/packages/nimble-blazor/)** - Handy bindings to the Spright components for Blazor applications.
> - **[`@ni/spright-components`](/packages/spright-components/)** - Spright components to be used by [any  type of application](https://custom-elements-everywhere.com/).

Author: ```suggestion
- [`@ni/spright-angular`](/angular-workspace/projects/ni/spright-angular/) - Handy bindings to the Spright components for Angular applications.
- [`@ni/spright-blazor`](/packages/nimble-blazor/) - Handy bindings to the Spright components for Blazor applications.
- [`@ni/spright-components`](/packages/spright-components/) - Spright components to be used by [any  type of application](https://custom-elements-everywhere.com/).
```

The utility packages:

[![Nimble Tokens NPM version and repo link](https://img.shields.io/npm/v/@ni/nimble-tokens.svg?label=@ni/nimble-tokens)](https://www.npmjs.com/package/@ni/nimble-tokens)
[![Jasmine parameterized NPM version and repo link](https://img.shields.io/npm/v/@ni/jasmine-parameterized.svg?label=@ni/jasmine-parameterized)](https://www.npmjs.com/package/@ni/jasmine-parameterized)
[![XLIFF to JSON Converter for Angular NPM version and repo link](https://img.shields.io/npm/v/@ni/xliff-to-json-converter.svg?label=@ni/xliff-to-json-converter)](https://www.npmjs.com/package/@ni/xliff-to-json-converter)

> - **[`@ni/nimble-tokens`](/packages/nimble-tokens/)** - Base design tokens used by the Nimble and Spright component packages.

Author: ```suggestion
- [`@ni/nimble-tokens`](/packages/nimble-tokens/) - Base design tokens used by the Nimble and Spright component packages.
```

### Nimble packages

The general-use Nimble packages implementing the Nimble Design System.

[![Nimble Angular NPM version and repo link](https://img.shields.io/npm/v/@ni/nimble-angular.svg?label=@ni/nimble-angular)](https://www.npmjs.com/package/@ni/nimble-angular)
[![Nimble Blazor Nuget version and repo link](https://img.shields.io/nuget/v/NimbleBlazor.svg?label=NimbleBlazor)](https://www.nuget.org/packages/NimbleBlazor)
[![Nimble Components NPM version and repo link](https://img.shields.io/npm/v/@ni/nimble-components.svg?label=@ni/nimble-components)](https://www.npmjs.com/package/@ni/nimble-components)

- **[`@ni/nimble-angular`](/angular-workspace/projects/ni/nimble-angular/)** - Handy bindings to the Nimble components for Angular applications.
- **[`@ni/nimble-blazor`](/packages/blazor-workspace/)** - Handy bindings to the Nimble components for Blazor applications.
- **[`@ni/nimble-components`](/packages/nimble-components/)** - Nimble components to be used by [any type of  application](https://custom-elements-everywhere.com/).

### Spright packages

> The context-specific Spright packages used for specific applications.

Reviewer: This feels a little vague – how about stealing the first line from the Spright readme?
```suggestion
NI-styled web components following [Nimble Design System](https://nimble.ni.dev/) patterns but not part of the core design system.
```

Author: I don't think that particular description helps a user that much either. It just says it's stuff not in Nimble design system which is guessable from being a separate package and it's not really clear what "core" means.

I think a clear description would say what it is instead of what it isn't/

```suggestion
The Spright packages contain components that are built using Nimble technology and design patterns but tailored for specific use-cases or applications instead of being general purpose.
```

[![Spright Angular NPM version and repo link](https://img.shields.io/npm/v/@ni/spright-angular.svg?label=@ni/spright-angular)](https://www.npmjs.com/package/@ni/spright-angular)
[![Spright Blazor Nuget version and repo link](https://img.shields.io/nuget/v/SprightBlazor.svg?label=SprightBlazor)](https://www.nuget.org/packages/SprightBlazor)
[![Spright Components NPM version and repo link](https://img.shields.io/npm/v/@ni/spright-components.svg?label=@ni/spright-components)](https://www.npmjs.com/package/@ni/spright-components)

> - [`@ni/spright-angular`](/angular-workspace/projects/ni/spright-angular/) - Handy bindings to the Spright components for Angular applications.

Reviewer: nit: The tone is friendly, but 'handy' might be too casual - it also makes me think of https://www.handy.com

```suggestion
- [`@ni/spright-angular`](/angular-workspace/projects/ni/spright-angular/) - Angular bindings for Spright components.
```

Author: I was trying to convey a tone of "these are handy / useful but technically not strictly necessary".
I can go with the straightforward message if preferred. Would the less colloquial "useful" be a good alternate?

```suggestion
- [`@ni/spright-angular`](/angular-workspace/projects/ni/spright-angular/) - Useful bindings to the Spright components for Angular applications.
```
